### DISCUSSION TITLE
infinispan-config-14.0.xsd seems invalid

### DISCUSSION THREAD

**Discussion by synth3** [infinispan/infinispan#10695] [Q&A] [answered]
Hello,

while trying to update my infinispan configuration from 13.x to 14.x I noticed that https://infinispan.org/schemas/infinispan-config-14.0.xsd is reported as invalid:

```
The content of 'query' must match (annotation?, (simpleType | complexType)?, (unique | key | keyref)*)). A problem was found starting at: attribute.

Content of type 'integer' is expected. The following content is not a valid type: '${Sites.max-cleanup-interval}'

Content of type 'boolean' is expected. The following content is not a valid type: '${Backup.enabled}'
```

Might I be doing something wrong or is there an updated XSD available?

**Comment by tristantarrant** [comment]
Unfortunately we published the source file instead of the filtered one. I should have that fixed by today

**Reply by synth3** [reply]
Thanks alot! Would that also solve the `The content of 'query' must match [...] A problem was found starting at: attribute.`?

**Comment by tristantarrant** [comment, marked as answer]
I pushed the new schemas

**Reply by synth3** [reply]
Thank you! :)
